src/wp-includes/functions.php

```diff
@@ -3823,7 +3823,7 @@ function absint( $maybeint ) {
  * @access private
  *
  * @param string $function    The function that was called.
- * @param string $version     The version of WordPress that deprecated the function.
+ * @param string $version     The version of ClassicPress that deprecated the function.
```

Comment:
Let's say "ClassicPress or WordPress version number" for now.  We'll prefix these version numbers with `WP-` if they are WordPress versions.  See #27.